### PR TITLE
Adding logos and support

### PR DESCRIPTION
**Pull request by kadrlica**
* Adding LSSTC and institutional logos
* Adding some text on support
* Adding separate code of conduct page
* Including "Rubin Observatory" in some places

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -278,29 +279,30 @@ <h2>Code of Conduct</h2>
         with this code of conduct are expected to comply immediately.</p>
       <p>If a participant engages in behavior that violates this code of conduct, the organizers may take action they
         deem appropriate, including warning the offender or expulsion from the group.</p>
-      <hr class="footnotes-sep">
 
+      <p>This code of conduct is derived from the
+        <a href="http://lsstdesc.org/sites/default/files/LSST_DESC_Professional_Conduct.pdf" target="_blank">Meeting Code of Conduct of the LSST Dark Energy Science Collaboration</a> and the <a href="https://us.pycon.org/2018/about/code-of-conduct/" target="_blank">PyCon Code of Conduct</a>. More on our code of conduct can be found <a href="conduct.html">here</a>.</p>
```
Comment (yymao) [inline index.html]:
```suggestion
        <a href="http://lsstdesc.org/sites/default/files/LSST_DESC_Professional_Conduct.pdf" target="_blank">Meeting Code of Conduct of the LSST Dark Energy Science Collaboration</a> and the <a href="https://us.pycon.org/2018/about/code-of-conduct/" target="_blank">PyCon Code of Conduct</a>.</p>
```

File: index.html [hunk 2]
```diff
@@ -261,9 +262,9 @@ <h2>Code of Conduct</h2>
       <p>By joining the group you agree to:</p>
       <ul>
         <li>
-          <p>Behave professionally. Harassment<sup>[1]</sup>, and exclusionary or offensive
+          <p>Behave professionally. Harassment, and exclusionary or offensive
```
Comment (yymao) [inline index.html]:
```suggestion
          <p>Behave professionally. <span class="note" title="Harassment includes, but is not limited to, sustained disruption of talks or other events, unwelcome physical contact, sexual attention or innuendo, intimidation, stalking, and recording of an individual without consent.">Harassment</span>, and exclusionary or offensive
```

File: index.html [hunk 3]
```diff
@@ -261,9 +262,9 @@ <h2>Code of Conduct</h2>
...
             comments or jokes related
-            to personal attributes<sup>[2]</sup> are not appropriate.</p>
+            to personal attributes are not appropriate.</p>
```
Comment (yymao) [inline index.html]:
```suggestion
            to <span class="note" title="Personal attributes include, but are not limited to, (alphabetically) age, disability, ethnicity, gender, gender expression, gender identity, lactation, nationality, physical appearance, political affiliation, pregnancy, race, religion, sexual orientation, and status as a caregiver (including as a parent).">personal attributes</span> are not appropriate.</p>
```

File: index.html [hunk 4]
```diff
@@ -88,7 +88,7 @@
```
Comment (yymao) [inline index.html]:
```suggestion

    span.note {
      cursor: help;
      border-bottom: 1px dashed #333;
    }

```

File: conduct.html [hunk 5]
```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
```
Comment (yymao) [inline conduct.html]: Remove this page.

**Review by yymao — APPROVED** [review]
Thanks for the PR @kadrlica. Can we put the notes for the CoC directly inline instead of on a new page? See my suggestions. Otherwise this looks good to me.

**Comment by kadrlica** [comment]
I'm fine with including the text in the index page, but I didn't like the way the footnotes were being formatted at the botton. I don't know how to pull suggestions to test them without committing, so I'll commit your suggestions, test them out, and maybe submit other changes.

**Comment by kadrlica** [comment]
I like the mouse-over notes. Looks good to me. Thanks!

**Pull request merged**